src/api/Controllers/ImportController.cs

```diff
@@ -120,6 +158,63 @@ public async Task<ActionResult<int>> UploadJsonFileAsync(int workgroupId, IFormF
         }
     }
 
+    private async Task UploadAttachments(IFormFile boreholesFile, List<BoreholeImport> boreholes)
+    {
+        foreach (var borehole in boreholes.Select((value, index) => (value, index)))
+        {
+            if (borehole.value.BoreholeFiles != null && borehole.value.BoreholeFiles.Count > 0)
+            {
+                using var zipStream = boreholesFile.OpenReadStream();
+                using var zipArchive = new ZipArchive(zipStream, ZipArchiveMode.Read);
```

Comment:
REQ: Ich denke es wäre gut den StreamReader aus dem Loop zu nehmen.

```suggestion
    {
       using var zipStream = boreholesFile.OpenReadStream();
       using var zipArchive = new ZipArchive(zipStream, ZipArchiveMode.Read);
        foreach (var borehole in boreholes.Select((value, index) => (value, index)))
        {
            if (borehole.value.BoreholeFiles != null && borehole.value.BoreholeFiles.Count > 0)
            {
```